src/lib.rs

```diff
@@ -198,6 +198,18 @@
 //!     #[error("Oops it blew up")]
 //!     #[diagnostic(code(my_lib::bad_code))]
 //!     BadThingHappened,
+//!
+//!     #[error(transparent)]
```

Comment:
Shouldn't `#[diagnostic(transparent)]` imply `#[error(transparent)]`?

Reply:
miette is blissfully unaware of thiserror. They are completely separate macros, and miette doesn't actually do anything related to `std::error::Error` itself.

Reply:
@zkat If you're using darling (https://github.com/TedDriggs/darling) you can parse `#[error]` attributes and infer settings from it. This would help to avoid all the duplicate info in some cases.